install_clojure_lsp

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Install the latest version of an executable file from a Github release
+# Keep a local copy of old versions
+# Requires curl
+
+if [ "$(id -u)" != "0" ]; then
+  echo "This script must be run as root" 1>&2
+  exit 1
```

Comment:
I would prefer not having a script that must be run as root, if the `DST_DIR` could be specified and/or defaulted to `$HOME/bin` I think the rest of the script could be run without root. 

What are your thoughts? Is that too paranoid or weird for install scripts?